app/html/show.html

```diff
@@ -14,26 +13,29 @@
         <thead>
             <caption>Show Files:</caption>
             <tr>
-                <th onclick="sortTableCheckbox(0)">Selected</th>
-                <th onclick="sortTable(1)">Name</th>
+                <th>Element</th>
+                <th>Audio</th>
             </tr>
         </thead>
         <tbody id="tableBody">
         </tbody>
     </table>
     <button id='addShow'>Add element</button>
+    <br><br>
+    <button id='runShow' value='notPlaying'>Play</button>
 </body>
-    <script type="text/javascript" src="../js/addShow.js"></script>
-    <script type="text/javascript" src="../js/show.js"></script>
     <script>
         const button = document.getElementById('addShow');
         const iframe = window.parent.document.getElementById('frame');
         const header = document.getElementById('header');
-
-        button.onclick = function () {popWindow()};
-
+        let showElements = [];
+        let serialPort;
         var onlyShowName = iframe.value.split(path.sep);
```

Comment:
I'm getting a "path is not defined" for this. Forgotten variable, or reference to something in node? If the latter, this script stuff might need to get moved into a separate file (or scripts need to get re-ordered to get the reference in scope in time). Not modified through this PR, but still an issue